src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -526,6 +526,9 @@ protected override void Apply(ClusterOperationContext context, BlittableJsonRead
 
                     case nameof(DeleteServerWideTaskCommand):
                         var deleteConfiguration = UpdateValue<DeleteServerWideTaskCommand.DeleteConfiguration>(context, type, cmd, index, skipNotifyValueChanged: true);
+                        if(deleteConfiguration == null)
+                            throw new RachisInvalidOperationException($"The server wide task to delete doesn't exist: {cmd}");
```

Comment:
can this happened also for `ToggleServerWideTaskStateCommand` / `PutServerWideExternalReplicationCommand` / `PutServerWideBackupConfigurationCommand` ... etc
basically every command that use the return value of `UpdateValue`?

Reply:
I checked the other location and all of them handle it inside the `UpdateValueCommand<T>.GetUpdatedValue`.
I moved the current check to `DeleteServerWideTaskCommand.GetUpdatedValue` as well.